tests/unit/Body.ts

```diff
@@ -1,92 +1,59 @@
-import { assert } from 'chai';
-import { VNode } from '@dojo/interfaces/vdom';
-import WidgetBase from '@dojo/widget-core/WidgetBase';
-import WidgetRegistry from '@dojo/widget-core/WidgetRegistry';
-import * as registerSuite from 'intern/lib/interfaces/object';
-import { SinonSpy, spy, stub } from 'sinon';
-import { cleanProperties, spyOnWidget } from '../support/util';
+import * as registerSuite from 'intern!object';
+
```

Comment:
In the future we are likely to implement a rule about import ordering with alphabetization and blank lines can create separate "groups" where there should be alphabetized.  I personally believe it can improve readability and maintenance of code to break imports up into groups